jsoniter-scala/src/main/scala/com/ovoenergy/kafka/serialization/jsoniter_scala/package.scala

```diff
@@ -0,0 +1,3 @@
+package com.ovoenergy.kafka.serialization
+
+package object jsoniter_scala extends JsoniterScalaSerialization
```

Comment:
I am not sure if I like the `_`. @filosganga What do you think? Is camel-case any better?

Reply:
Or just drop `scala` suffix and make it `jsoniter`?

Reply:
Jsoniter is a name of Java/Go project that inspired me to reuse their ideas for the Scala world:
http://jsoniter.com/

Reply:
Snake case is recommended by official Java Tutorial: https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Using both upper and lower case chars in package names can lead for clashing on file systems that have case insensitive directories.

Reply:
Why don't we keep the module name `jsoniter`, so we don't have to handle this situation?